docs/dev-concepts/architectural-overview.md

```diff
@@ -73,63 +73,73 @@ To learn more, see [Content types](content-types).
-         <ul>
-            <li>Two participants can have multiple ongoing conversations using separate and multiple conversation topics.</li>
-            <li>The conversation topic name uses a random 32-byte alphanumeric string.</li>
-         </ul>
-      </td>
-      <td><code>m-XxBHrITqjd00nLMuTyaaGDvTLnviazU8</code>
-      </td>
-   </tr>
-   <tr>
-      <td><strong>Private store</strong>
-      </td>
-      <td>A client built with XMTP can choose to store its users' private key bundles locally or encrypted on the XMTP network. When required, a client uses a private store topic to store a user's private key bundle.
-      </td>
-      <td><code>privatestore-0x458dd9C5bf4d4E8f80Ba88923E3b94FD028CEe38</code>
-      </td>
-   </tr>
-</table>
+This section describes how topics and the message presentation flow work for the current version of the protocol, referred to as XMTP V2. Only client apps with XMTP client SDK >=v7.0.0 can use XMTP V2. To learn how topics and flows work in XMTP V1, see [XMTP V1 topics and message presentation flow](#xmtp-v1-topics-and-message-presentation-flow).
+
+In XMTP V2, clients use the following topics to perform their primary responsibilities.
+
+
+#### Contact topic
+
+A client uses a contact topic to advertise a user’s public key bundle on the XMTP network. The network advertises this public key bundle to enable other users to contact the user.
+
+The contact topic name uses this format: `contact-<account-address>`.
+
+For example, `contact-0x458dd9C5bf4d4E8f80Ba88923E3b94FD028CEe38`.
+
+To learn more, see [Contacts](https://github.com/xmtp/proto/blob/main/PROTOCOL.md#contacts) in the `xmtp/proto` repo.
+
+
+#### Invite topic
+
+Clients use invite topics to initiate conversations between participants. 
+
+The invite topic name uses this format: `invite-<account-address>`. 
+
+For example, `invite-0x458dd9C5bf4d4E8f80Ba88923E3b94FD028CEe38`.
+
+The client sends an invitation to both the sender and recipient’s invite topic. The invitation includes no message content, but includes:
+
+- A randomly generated conversation topic name, which tells clients which conversation topic to use to send and retrieve messages.
+
+- Encrypted key material, which includes a shared secret for message encryption.
+
+- Support for optional custom [conversation IDs](/docs/client-sdk/javascript/tutorials/filter-conversations#set-a-conversation-id) and other [metadata](/docs/client-sdk/javascript/tutorials/filter-conversations#set-conversation-metadata). Clients can use these IDs and metadata to [filter](/docs/client-sdk/javascript/tutorials/filter-conversations) and [label](/docs/client-sdk/javascript/tutorials/filter-conversations) conversations.
+
+To learn more, see [Invitations](https://github.com/xmtp/proto/blob/main/PROTOCOL.md#invitations) in the `xmtp/proto` repo.
+
+
+#### Conversation topic
+
```

Comment:
Having every app have their own conversationId would make the universal inbox a lot less universal. When I am talking to you on both Converse and xmtp.chat, why would I want that to be split into two conversations?

Reply:
+1, the default for new apps should be `null` convoId. 

If there's added value to the user, other convoIds can be used to create new conversations between the same peers with the convoId as the immutable context for that conversation. 

It may be helpful to draw a parallel to the 'Subject' in emails that adds context to the email. ConversationIds do the same for XMTP conversations. 

Lastly, may be worth calling out that using convoIds to provide info about the client about of origin (e.g. Converse, Orb, etc) isn't verifiable and usually creates more friction (by creating multiple conversations artificially) than the value it adds.